### PR TITLE
[py-torch-fidelity] New package

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-torch-fidelity/package.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    homepage = "https://www.github.com/toshas/torch-fidelity"
+    pypi = "torch_fidelity/torch_fidelity-0.3.0.tar.gz"
+
+    license("Apache-2.0", checked_by="qwertos")
+
+    version("0.3.0", sha256="3d3e33db98919759cc4f3f24cb27e1e74bdc7c905d90a780630e4e1c18492b66")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("pil", type=("build", "run"))
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-torch-fidelity/package.py:23]: I think you mean `py-pillow` here (according to the staged `requires.txt`):

```suggestion
    depends_on("py-pillow", type=("build", "run"))
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed the homepage, version sha256 and all but one dependency.

File: var/spack/repos/builtin/packages/py-torch-fidelity/package.py (hunk 1 above)
Comment (qwertos) [reply var/spack/repos/builtin/packages/py-torch-fidelity/package.py:23]: Is replacing all `py-pillow` dependencies with the provider `pil` not a thing anymore?

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/py-torch-fidelity/package.py:23]: I'm new to checking python package PR dependencies so wasn't aware it was a thing.  @adamjstewart ?

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/py-torch-fidelity/package.py:23]: Nvm.  I grep'd python packages and see `pil` is a virtual provided by `py-pillow`.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**